enos/enos-scenario-autopilot.hcl

```diff
@@ -141,6 +151,7 @@ scenario "autopilot" {
       vault_init                  = false
       vault_license               = step.read_license.license
       vault_local_artifact_path   = local.bundle_path
+      vault_artifactory_release   = local.install_artifactory_artifact ? step.build_vault.vault_artifactory_release : null
```

Comment:
I assume the other build modules will always return null for the artifactory release so we can probably always use that output.

Reply:
The other modules do not return that output, so might error out if I don't condition it here.